build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -131,10 +143,12 @@ private static bool CheckIfAllBuildsHavePublished()
                  { "Windows_x86", false },
                  { "Windows_x64", false },
                  { "Ubuntu_x64", false },
+                 { "Ubuntu_16_04_x64", false },
                  { "RHEL_x64", false },
                  { "OSX_x64", false },
                  { "Debian_x64", false },
-                 { "CentOS_x64", false }
+                 { "CentOS_x64", false },
+                 { "Fedora_23_x64", false }                 
```

Comment:
We need to change L156 to call Monikers.GetBadgeMoniker() instead of producing it itself.